runner/README.md

```diff
@@ -76,6 +78,29 @@ The `local` backend will submit the run to the locally started shim and runner.
 Continue? [y/n]:
 ```
 
+## Testing remotely
+
+You can also test the built shim and runner using standard backends (including SSH fleets).
+
+> [!NOTE]
+> To run with standard backends, both the runner and shim must be built for linux/amd64.
+
+Build the runner and shim, and upload them to S3 automatically using `just` (see [justfile](../justfile)).
+
+```shell
+just upload
```

Comment:
How should external contributors use it if they don't have access to dstack-runner-downloads-stgn?

Reply:
I will extract the bucket name variable and document it.